src/Git/CheckoutBranchViaConsole.php

```diff
@@ -0,0 +1,18 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Laminas\AutomaticReleases\Git;
+
+use Symfony\Component\Process\Process;
+
+class CheckoutBranchViaConsole implements CheckoutBranch
+{
+    public function __invoke(
+        string $repositoryDirectory,
+        string $branchName
+    ): void {
+        (new Process(['git', 'checkout', $branchName], $repositoryDirectory))
+            ->mustRun();
```

Comment:
I wonder if we should use `switch` these days

Reply:
Yes, we should. I'll update here, and elsewhere.

Reply:
Actually... will only update in the stuff relevant to this particular patch. I'll do a separate PR to update this everywhere later.